Mathlib/RingTheory/PowerSeries/Inverse.lean

```diff
@@ -368,10 +369,11 @@ instance : NormalizationMonoid k⟦X⟧ where
     exact ((eq_divided_by_X_pow_order_Iff_Unit h₀.ne_zero).mpr h₀).symm
 
 theorem normUnit_X : normUnit (X : PowerSeries k) = 1 := by
-  simp [normUnit, ← Units.val_eq_one, Unit_of_divided_by_X_pow_order_nonzero]
+  simp only [normUnit, inv_eq_one, ← Units.val_eq_one, ne_eq, X_ne_zero, not_false_eq_true,
```

Comment:
This feels significantly less readable - I'd only like to make this change with good justification.